andela/Ah-frontend-guardians
# feature(view-profiles): Users can view and edit profiles

### What does this PR do?
A user should be able to view and edit his profile

### Description of tasks to be completed

* Implement view and editing a profile
* Design the user interface for performing the functionality

### How should this be manually tested?
Use the appropriate screenshots provided to view the expected results
* Visit the url below:
https://ah-frontend-guardians-pr-17.herokuapp.com

* signup and login
* click on the avatar profile avatar to view your profile page.
* Click the edit button if you want to make changes to your profile page
* You can make changes to your `first name`, `last name`, `image` or `bio` fields.

### What are the relevant pivotal tracker stories?
[#162685627](https://www.pivotaltracker.com/story/show/162685627)

### View a profile page
<img width="1118" alt="screen shot 2019-02-27 at 7 59 18 am" src="https://user-images.githubusercontent.com/30922279/53467127-d61f6380-3a65-11e9-8e74-af2b490d1f4b.png">

### Edit a profile page
<img width="1118" alt="screen shot 2019-02-27 at 8 01 46 am" src="https://user-images.githubusercontent.com/30922279/53467148-f3ecc880-3a65-11e9-93a4-22f185b299ce.png">
